### PR TITLE
Add Ukrainian Localization

### PR DESCRIPTION
**Pull request by AndrejGorodnij**
I'd added 🇺🇦 Ukrainian Localization.

Thanks for this great project! Слава Україні!

### CODE REVIEW

File: auto_py_to_exe/web/js/i18n.js [hunk 1]
```diff
@@ -781,6 +819,9 @@ const translationMap = {
                 zh_tw: '使用單檔案模式的附加檔案時要小心;\n' +
                     '<a href="https://stackoverflow.com/a/13790741/" style="text-decoration: none;">read this</a>\n' +
                     '並更新您的代碼以使用 PyInstaller。',
+                ua: 'Будьте уважні під час використання додаткових файлів в режимі onefile;' +
+                '<a href="https://stackoverflow.com/a/13790741/" style="text-decoration: none;">Прочитайте це</a>\n' +
+                'і оновіть свій код для подальшого використання PyInstaller.',
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:824]: Just a small indentation fix: 

```suggestion
                ua: 'Будьте уважні під час використання додаткових файлів в режимі onefile;' +
                    '<a href="https://stackoverflow.com/a/13790741/" style="text-decoration: none;">Прочитайте це</a>\n' +
                    'і оновіть свій код для подальшого використання PyInstaller.',

```

**Review by brentvollebregt — COMMENTED** [review]
Looking really good! Just one small fix for i18n.js

Can you also add a line to the [translation table in the README file](https://github.com/brentvollebregt/auto-py-to-exe/blob/master/README.md?plain=1#L106)?

File: auto_py_to_exe/web/js/i18n.js (hunk 1 above)
Comment (AndrejGorodnij) [reply auto_py_to_exe/web/js/i18n.js:824]: Yep, just a second

**Comment by brentvollebregt** [comment]
Sweet, I see you updated the README, could you also get that indentation fix in i18n.js?

**Comment by AndrejGorodnij** [comment]
Oh, sorry, I missed that, thought You had already did it in my PR somehow, during the comment 🤷‍♂️ Sorry, my bad 🙃

**Review by brentvollebregt — APPROVED** [review]

**Comment by brentvollebregt** [comment]
v2.29.0 has just been release to PyPI with this change - thank you!

**Pull request merged**